src/app/fizzbuzz/fizzbuzz.component.ts

```diff
@@ -0,0 +1,76 @@
+import { FizzBuzzModel } from './../core/models/fizz-buzz.model';
+import { applicationConstants } from './../core/constants/application-constants';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-fizzbuzz',
+  templateUrl: './fizzbuzz.component.html',
+  styleUrls: ['./fizzbuzz.component.scss'],
+})
+export class FizzbuzzComponent implements OnInit {
+  public fizzBuzzArray: FizzBuzzModel[] = [];
+  public errorMessage: string = '';
+  public fizzBuzzInput!: string;
+  public fizzBuzzValue!: FizzBuzzModel;
+
+  ngOnInit(): void {
+    this.fillFizzArray();
+  }
+
+  private fillFizzArray(): void {
+    for (let i = 0; i < 100; i++) {
+      const element = this.fizzBuzz(i);
+      this.fizzBuzzArray.push({ element, isContiguous: this.isContiguous(i) });
+    }
+  }
+
+  private isContiguous(number: number): boolean {
```

Comment:
la spec n'était peut etre pas assez explicite. L'objectif était de trouver les nombres consécutifs. 
Cette implémentation y répond mais c est dans l'affichage où on ne sait pas qu'en rentrant '5' qui retourne true si au final 5/6 sont contigues ou bien 4/5. 

car on ne voit que 5 => Buzz => Is contiguous une fois affiché à l'écran